CONTRIBUTING.md

```diff
@@ -337,6 +337,10 @@ The exercise should consist of, at minimum:
 * A test suite
 * A reference solution that passes the test (see [#reference-solution](#reference-solution))
 
+You will need to add the exercise to `"exercises"` section of the `config.json` file in the track.
+The order in which the exercises are listed there is the order in which they are fetched by default by `exercism fetch`.
+Typically, exercises are ordered by difficulty, unless there is a particular reason to do otherwise.
```

Comment:
I am hoping this sentence doesn't conflict with any track that may wish to specify a different ordering? I think it is true that in the steady state all tracks want to order their exercises by difficulty, but I seem to remember some track that requested its contributors put new exercises at the end of the track temporarily, and they'll move them to the right spot later after seeing how difficult it seems for students. But alas, I didn't note down what track that was and I do not seem to be able to find it...